recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pycld2" %}
+{% set version = "0.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+
+test:
+  imports:
+    - pycld2
+  commands:
+    - pip check
+  requires:
+    - pip
+    - wheel==0.23.0
```

Comment:
Any reason this can't just be `wheel`?

Reply:
It's stange for me too but the requirement file in the project's repository got only wheel in a specific version https://github.com/aboSamoor/pycld2/blob/master/requirements.txt

Reply:
Trying with `wheel >= 0.23.0`